frontend/src/configs.js

```diff
@@ -41,9 +34,22 @@ export const URL_COMM_SVC_READ_CHAT = URL_COMM_SVC + PREFIX_COMM_SVC_READ_CHAT;
 /////////////// Collaboration Service ///////////////
 // Socket
 export const URL_COLLAB_SVC_SOCKET = URI_SVC;
-export const PATH_COLLAB_SVC_SOCKET = '/api/collaboration-service/socket';
+export const PATH_COLLAB_SVC_SOCKET = '/api/collab-service/socket';
 
 /////////////// Question Service ///////////////
 // API
 const PREFIX_QUES_SVC = '/api/question-service';
 export const URL_QUES_SVC = URI_SVC + PREFIX_QUES_SVC;
+
+/////////////// Auth Service ///////////////
+// API
+const PREFIX_AUTH_SVC = '';

```

Comment:
Ermm sorry I might miss this out but what's the point of empty string here?

Reply:
Added this initially for consistency sake but I shall remove it since it is not used anyway.